app/models/votes.rb

```diff
@@ -0,0 +1,11 @@
+class Votes < ActiveRecord::Base
+  belongs_to :api
+  belongs_to :user
+
+  def self.calculate_votes(api_id)
+    @api = Api.find(api_id)
+    @upvotes = Votes.where(user_vote: true, api: @api).count
+    @downvotes = Votes.where(user_vote: false, api: @api).count
+    { upvotes: @upvotes, downvotes: @downvotes }
```

Comment:
Great work with this method, but now you need to add some additional ones to do the other work being done in your controller!